wp-content/plugins/jetpack/_inc/build/comment-likes/comment-like-count.min.js

```diff
@@ -0,0 +1 @@
+jQuery(document).ready((function(t){var e=[];t(".comment-like-count").each((function(){var n=t(this).attr("data-blog-id"),a=t(this).attr("data-comment-id");e.push("/sites/"+n+"/comments/"+a+"/likes")})),t.ajax({type:"GET",url:"https://public-api.wordpress.com/rest/v1/batch",dataType:"jsonp",data:"urls[]="+e.map(encodeURIComponent).join("&urls[]="),success:function(e){for(var n in e)if(!e[n].error_data){var a=n.split("/")[4],o=e[n].found;if(o<1)return;t("#comment-like-count-"+a).find(".like-count").hide().text(o).fadeIn()}},error:function(){}})}));
```

Comment:
:warning: **Warning( severity 5 )**: File appears to be minified and cannot be processed (*Internal.Tokenizer.Exception*).